src/fides/api/schemas/connection_configuration/connection_secrets_scylla.py

```diff
@@ -31,7 +31,7 @@ class ScyllaSchema(ConnectionConfigSecretsSchema):
     )
     keyspace: Optional[str] = Field(
         title="Keyspace",
-        description="The keyspace used.",
+        description="The keyspace used. If not provided, DSRs for this integration will error. ",
```

Comment:
Yeah, this was a bit annoying.  

So Scylla doesn't seem to have a "default" keyspace, unlike our SQL integrations (e.g in Postgres we have the `public` schema). So the following query 
`SELECT user_id FROM users WHERE email = 'test@test.com';` will work in a SQL integration but not in Scylla, cause there's no keyspace specified. The way to do this in Scylla is to either call `USE the_keyspace_name;` beforehand, or use the full name of the table in the query `SELECT user_id FROM the_keyspace_name.users WHERE email = 'test@test.com';`  So we NEED the user to tell us what the keyspace is.

Your guess is correct, for D & D we don't need to set a specific keyspace since it will just discover them. 

I think that ideally we wouldn't need to set the keyspace at the integration level (we had quite a long slack thread on this the other day, I'll slack it to you), since this is effectively preventing us from linking multiple schemas (or keyspaces in Scylla) to a single system integration. 

I'm happy to discuss more if you think there's a better way to solve this , and I agree about adding the D & D scenario in the description so it's clearer for users.